ruby/lib/minitest/queue/order_reporter.rb

```diff
@@ -4,21 +4,28 @@
 class Minitest::Queue::OrderReporter < Minitest::Reporters::BaseReporter
   def initialize(options = {})
     @path = options.delete(:path)
+    @file = nil
     super
   end
 
   def start
-    @file = File.open(@path, 'w+')
-    super
+    file.truncate(0)
```

Comment:
Is `#start` called only once before forking?

If not, is there a risk that you truncate the few first logs in the case the first tests ran very quickly?

Also, is it okay not to call `super`?

Reply:
Yes it is called once before forking, so the truncate will behave like opening the file in write mode and rewriting sections of it in a non-forked setup.

Reply:
Looks like we call https://github.com/minitest/minitest/blob/7c907429e8e7ec7c3b6ee0c7045e67249f4ad505/lib/minitest.rb#L864

Probably not needed, I'll remove it.